source/welcome/insights.rst

```diff
@@ -4,17 +4,17 @@ Insights
 .. include:: ../_static/badges/allplans-cloud-selfhosted.rst
   :start-after: :nosearch:
 
+.. important::
+
+  From Mattermost v8.0, Insights has been disabled for all new instances and for existing servers that upgrade to Mattermost v8.0. The Insights feature flag can be re-enabled with an environment variable via ``MM_FEATUREFLAGS_INSIGHTSENABLED=true``.
```

Comment:
John mentioned that we don't want to mention the environment variable flag in the docs.